### PR TITLE
Support `cpu.family` hardware requirement for `mrack`

### PR DESCRIPTION
**Pull request by skycastlelily**
Pull Request Checklist

* [x] implement the feature
* [x] write the documentation
* [x] extend the test coverage
* [x] update the specification
* [ ] adjust plugin docstring
* [ ] modify the json schema
* [x] mention the version
* [ ] include a release note

### CODE REVIEW

File: tmt/steps/provision/mrack.py [hunk 1]
```diff
@@ -229,6 +229,18 @@ def _transform_beaker_pool(
         actual_value)
 
 
+def _transform_cpu_family(
+        constraint: tmt.hardware.NumberConstraint,
```
Comment (happz) [inline tmt/steps/provision/mrack.py:233]: Should be `IntegerConstraint`, CPU family will never be a floating point number.

**Comment by skycastlelily** [comment]
Yeah, this one is not based on cpu-frequency,so it didn't know about
IntegerConstraint,now
rebased, and updated^^

**Comment by skycastlelily** [comment]
Yeah, this one is not based on cpu-frequency,so it didn't know about IntegerConstraint,now rebased, and updated^^

**Comment by happz** [comment]
@skycastlelily I think the missing piece is a note in release notes & an update for HW requirements stating `cpu.family` is now supported by `beaker` plugin.

**Review by martinhoyer — APPROVED** [review]

**Review by thrix — APPROVED** [review]
LGTM with the requested docs change

**Review by therazix — APPROVED** [review]

**Comment by psss** [comment]
Failing `el9` job is irrelevant to this, merging.

**Pull request merged**